resources/views/projects/create.blade.php

```diff
@@ -93,35 +83,34 @@
 			</select>
 		</div>
 		<!-- Skillset -->
-		<div class="field">
+		<div class="field {{ $errors->has('skillsets') ? 'error': '' }}">
 			<label for="skillsets">Skillsets</label>
-			<select name="skillsets[]" multiple="" class="ui fluid dropdown">
+			<select name="skillsets" multiple="" class="ui fluid dropdown">
```

Comment:
sorry for being annoying, pero siguen saliendo mal algunas cosas 🙈 
ex: debería ser `name="skillsets[]"` pero aquí sale `name="skillsets"` (y así en varias líneas)